.github/ISSUE_TEMPLATE.md

```diff
@@ -29,3 +29,5 @@ You do NOT have to include this information if this is a FEATURE REQUEST
 
 **- Please mention your CMS, node.js, and operating system version.**
 <!-- You can find the CMS version by checking your web browser's developer tools console while in the CMS. -->
+
+**- Please link or paste your `config.yml` below if applicable.**
```

Comment:
Just one thing, it always makes me wonder a little bit when an issue template says "if applicable", unless they explain when it is needed in more detail.

Reply:
Yeah, I'm just worried that people who don't have (or aren't aware of) the `config.yml` may decide not to file an issue if they think this is required. I'd welcome suggestions for rephrasing this.

Reply:
"if possible"? that sounds pretty weird, though.

Maybe for the last two questions, we should put something to the effect of: "The following information is not required, but please provide as much as possible to help us reproduce your issue."